### PR TITLE
chore: update common dependency with cluster Config & restart host operator

### PR DESCRIPTION
**Pull request by MatousJobanek**
related PR codeready-toolchain/toolchain-common#217

it also restarts host operator so it can get the created ToolchainCluster CRs

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -35,4 +35,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/MatousJobanek/toolchain-common v0.0.0-20211119160239-6df56eb6f7e0
```
Comment (alexeykazakov) [inline go.mod:38]: Reminder

**Review by alexeykazakov — APPROVED** [review]

**Review by rajivnathan — APPROVED** [review]

File: make/test.mk [hunk 2]
```diff
@@ -109,11 +109,13 @@ print-operator-logs:
 
 .PHONY: setup-toolchainclusters
 setup-toolchainclusters:
-	echo ${MEMBER_NS_2}
 	$(MAKE) run-cicd-script SCRIPT_PATH=scripts/add-cluster.sh  SCRIPT_PARAMS="-t member -mn $(MEMBER_NS) -hn $(HOST_NS) -s"
 	if [[ ${SECOND_MEMBER_MODE} == true ]]; then $(MAKE) run-cicd-script SCRIPT_PATH=scripts/add-cluster.sh  SCRIPT_PARAMS="-t member -mn $(MEMBER_NS_2) -hn $(HOST_NS) -s -mm 2"; fi
 	$(MAKE) run-cicd-script SCRIPT_PATH=scripts/add-cluster.sh  SCRIPT_PARAMS="-t host   -mn $(MEMBER_NS)   -hn $(HOST_NS) -s"
 	if [[ ${SECOND_MEMBER_MODE} == true ]]; then $(MAKE) run-cicd-script SCRIPT_PATH=scripts/add-cluster.sh  SCRIPT_PARAMS="-t host   -mn $(MEMBER_NS_2) -hn $(HOST_NS) -s -mm 2"; fi
+	echo "Restart host operator pods so it can get the ToolchainCluster CRs while it's starting up".
```
Comment (xcoulon) [inline make/test.mk:116]: isn't host-operator supposed to watch the ToolchainCluster CR?

Comment (MatousJobanek) [reply make/test.mk:116]: yes, but the manager is configured during the startup of the deployment
https://github.com/codeready-toolchain/host-operator/pull/542/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R179
so we ned to restart it when we do some change there

Comment (xcoulon) [reply make/test.mk:116]: ok, got it

**Review by xcoulon — APPROVED** [review]

**Pull request merged**